qcodes/instrument_drivers/Keysight/keysight_e4980a.py

```diff
@@ -386,6 +384,32 @@ def _set_measurement(self,
         self._measurement_pair = measurement_pair
         self.write(f":FUNCtion:IMPedance {measurement_pair.name}")
 
+    def _get_voltage_level(self) -> float:
+        """
+        Gets voltage level if signal is set with voltage level parameter
+        otherwise raises an error.
+        """
+        try:
+            v_level = self.ask(":VOLTage:LEVel?")
+        except Exception as e:
```

Comment:
Normally it is best not to capture general exceptions like this but only a specific type